src/moduler/filtrering/filter/type-filter.js

```diff
@@ -17,6 +17,7 @@ function TypeFilter({
             filter={aktivitetTyper}
             filterTittel="filter.aktivitet.type.tittel"
             filterTekst="aktivitet.type."
+            metrikkNavn="typefilter"
```

Comment:
Alle disse metrikknavn verdiene syns jeg skal ha mer beskrivende navn, for eksempel "aktivitetsplan.filtervalg.typefilter" eller kanskje at alle heter "aktivitetsplan.filternavn" også sende et objekt som beskriver hvilken type filter det gjelder